contracts/interchain-token-service-impl.clar

```diff
@@ -214,7 +214,7 @@
     (begin
         (asserts! (is-proxy) ERR-NOT-PROXY)
         (asserts! (get-is-started) ERR-NOT-STARTED)
-        (asserts!  (is-eq caller (get-owner)) ERR-NOT-AUTHORIZED)
+        (asserts! (is-eq caller (get-operator)) ERR-NOT-AUTHORIZED)
```

Comment:
Also modify `remove-trusted-address`

Reply:
@raress96 I’d keep `remove-trusted-address` as `owner-only` unless it's a strict requirement from the Axelar team since it has the power to cut off connected chains.